Kernel/BootInfo.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2021, Gunnar Beutner <gbeutner@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <Kernel/Arch/x86/PageDirectory.h>
+#include <Kernel/PhysicalAddress.h>
+#include <Kernel/VirtualAddress.h>
+
+extern "C" PhysicalAddress start_of_prekernel_image;
+extern "C" PhysicalAddress end_of_prekernel_image;
+extern "C" FlatPtr kernel_base;
+#if ARCH(X86_64)
+extern "C" u32 gdt64ptr;
+extern "C" u16 code64_sel;
+extern "C" PhysicalAddress boot_pml4t;
```

Comment:
iirc this should be pml4e?

Reply:
Wouldn't the `e` imply that it's an entry in the PML4 table (-> PML4T)?

Reply:
... Aaaaaaah. Yes. I never did get why there was an E.